my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
```

Comment:
Also, just so you're aware, there's a convention to write variables spaced with underscores, such as `col_1` rather than `col1`.